doc/method/method.md

```diff
@@ -1,19 +1,25 @@
 ## Taxonomy assembly overview
 
 Terminology: 
-
-  * polysemy = where a single name-string belongs to multiple taxon records; in 
-    nomenclatural terms, either a homonym or hemihomonym
-  * node = a taxon record derived from a source taxonomy, giving name-string,
-    parent taxon, and perhaps other information
+  
+  * union taxonomy = data structure for creation of the reference 
+    taxonomy
+  * source taxonomy = imported taxonomic source (NCBI taxonomy, etc.)
+  * node = a taxon record, either from a source taxonomy or the union taxonomy;
+    giving name-string, source information,
+    parent node, and perhaps other information such as rank
+  * polysemy = where a single name-string belongs to multiple nodes
+    (within the same taxonomy); in 
+    nomenclatural terms, either a homonym, hemihomonym, or mistaken 
+    clerical duplication
 
 The assembly process works, in outline, as follows:
 
  1. Start with an ordered list of imported source taxonomies S1, S2, ... (see above)
- 1. Initialize the 'union' taxonomy U to be empty
+ 1. Initialize the union taxonomy U to be empty
```

Comment:
I would write more about the skeleton, or give it more prominence, if I had a better name for it - 'skeleton' sounds kind of stupid. But I haven't thought of anything better yet.